API/CLI.md

```diff
@@ -81,11 +81,13 @@ During code deployment, CLI looks for `.tesselignore` and `.tesselinclude` files
   * `[--usb]` deploy over USB connection
   * `[--slim]` true by default, copy only files needed by the program to run
   * `[--full]` the opposite of --slim, copy all the files in the project directory
+  * `[--compress]` true by default, compress the project prior to running it
```

Comment:
It is probably more accurate to say that this action will "compress and minify the project code prior to copying it to the device". @rwaldron can confirm this.